forecasting/tests/unittests/test_forecasting.py

```diff
@@ -0,0 +1,34 @@
+from autogluon.forecasting import ForecastingPredictor
+from autogluon.forecasting import TabularDataset
+import autogluon.core as ag
+
+
+def test_forecasting():
+    train_data = TabularDataset("https://autogluon.s3-us-west-2.amazonaws.com/datasets/CovidTimeSeries/train.csv")
+    test_data = TabularDataset("https://autogluon.s3-us-west-2.amazonaws.com/datasets/CovidTimeSeries/test.csv")
```

Comment:
Can you subsample these datasets so the test runs quicker?

Reply:
I've tried before but I found that only reducing num_of_ts does not affect run time that much. Besides, this test should take less than 30s to finish as I set num_trials and epochs to be very small.